web/components/geo/map/FcMapPopupActionViewData.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <FcButton
```

Comment:
bit of a problem here: if we hover over a school (or other non-selectable feature), the `<v-card-actions>` part of the popup shows up as an awkward thin grey bar at bottom of the popup.

maybe move `SELECTABLE_LAYERS` et al. into `FcMapPopup` and use it as part of the `v-if` condition on `<v-card-actions>`?